packages/core/src/Cardano/ChainId.ts

```diff
@@ -1,11 +1,21 @@
 /**
- * Network type (mainnet or **some** testnet)
+ * The "network ID" identifies a network in the Cardano blockchain, it is included in every address and also
+ * optionally present in the transaction body.
+ *
+ * This can only store 16 possibilities (4 bits).
  */
 export enum NetworkId {
-  Mainnet = 1,
-  Testnet = 0
+  Mainnet = 0b0001,
```

Comment:
Isn't this the same as it was? :thinking:

Reply:
The change conflicts with the domain concept, which represents these as integers, so reverting is preferred.